pkg/releaser/git.go

```diff
@@ -157,6 +163,47 @@ func (r *Releaser) submitPR(request *source.ReleaseRequest) (string, error) {
 	return pr.GetHTMLURL(), nil
 }
 
+func (r *Releaser) closeExistingPR(request *source.ReleaseRequest, client *github.Client) error {
+	queryString := fmt.Sprintf("is:pr is:open author:%s repo:%s/%s release new version %s",
+		request.PluginReleaseActor,
+		r.UpstreamKrewIndexRepoOwner,
+		r.UpstreamKrewIndexRepo,
+		request.PluginName,
+	)
+	closeComment := fmt.Sprintf("Closing this PR as it's outdated\n\n/close")
+
+	existentPR, _, err := client.Search.Issues(
+		context.TODO(),
+		queryString,
+		nil,
+	)
+	if err != nil {
+		return err
+	}
+	if numPR := existentPR.GetTotal(); numPR > 0 {
+		logrus.Infof("Found %d outdated PRs, closing them before opening this new one\n",
+			numPR,
+		)
+		for _, pr := range existentPR.Issues {
+			logrus.Infof("Closing outdated PR #%d\n", pr.GetNumber())
+			_, _, err := client.PullRequests.CreateComment(
```

Comment:
The PullRequest.CreateComment is a review comment. You will have to use Issues.CreateComment instead to add comment to the PR.

Reply:
:heavy_check_mark: